tests/test_one_dim_qd.py

```diff
@@ -2,7 +2,11 @@
 import pytest
 import numpy as np
 
-from quantum_systems import ODQD, GeneralOrbitalSystem, SpatialOrbitalSystem
+from quantum_systems import (
+    ODSincDVR,
```

Comment:
This needs to be corrected to `ODQD` again.

Reply:
oh haha